rest/route/admin_banner.go

```diff
@@ -67,6 +67,12 @@ func (h *bannerGetHandler) Parse(ctx context.Context, r *http.Request) error {
 	return nil
 }
 
+// @Summary		Get the banner
+// @Description	Fetch the text and type of Evergreen's current banner
+// @Tags		info
+// @Router		/admin/banner [get]
+// @Security	Api-User || Api-Key
+// @Success		200	{object}	model.APIBanner
```

Comment:
Is it possible to add other error types and explain what they mean or nah?

Reply:
(not here specifically, just conceptually)

Reply:
Might just be good to document (wherever we document this) how to do that so we can consider it for the future.

Reply:
Or yeah maybe a follow up ticket.